tests/rule_based_profiler/domain_builder/test_cardinality_checker.py

```diff
@@ -83,7 +83,8 @@ def test_cardinality_checker_instantiation_valid_max_proportion_unique_parameter
 
 def test_cardinality_checker_instantiation_invalid_limit_mode_parameter():
     with pytest.raises(ProfilerConfigurationError) as excinfo:
-        _: CardinalityChecker = CardinalityChecker(
+        # noinspection PyUnusedLocal,PyTypeChecker
+        cardinality_checker: CardinalityChecker = CardinalityChecker(
             limit_mode="&^*%not_a_valid_mode^&*("
         )
```

Comment:
Since we are not doing assignment here (and in all the other examples) maybe it's cleaner to just remove the assignment altogether e.g.:
```suggestion
        CardinalityChecker(limit_mode="&^*%not_a_valid_mode^&*(")
```

Reply:
@anthonyburdi I find it nice to indicate that "the thing" returns something.  So the choice for me was to use `_ =` without a type hint or a dummy return value with a type hint -- I opted for the latter in the name of interpretability.  If you feel strongly in favor of a different way, please let me know, and I will change it.  Thanks!